CMakeLists.txt

```diff
@@ -46,9 +32,82 @@ set(INEXOR_APP_VERSION_MAJOR 0)
 set(INEXOR_APP_VERSION_MINOR 1)
 set(INEXOR_APP_VERSION_PATCH 0)
 
-find_package(Git REQUIRED)
+find_package(Vulkan)
+
+include(FetchContent)
+set(FETCHCONTENT_QUIET OFF)
+
+FetchContent_Declare(fmt
+    GIT_REPOSITORY https://github.com/fmtlib/fmt.git
+    GIT_TAG 8.0.1
+    GIT_SHALLOW ON
+    GIT_PROGRESS ON)
+FetchContent_MakeAvailable(fmt)
```

Comment:
We only need fmt if we configure spdlog with `SPDLOG_FMT_EXTERNAL` otherwise spdlog ships with its own fmt.

Reply:
I removed that.